tests/test_benchmark.py

```diff
@@ -280,7 +400,199 @@ def test_hif2a():
     benchmark_hif2a(verbose=True, num_batches=2, steps_per_batch=100)
 
 
+def test_ixn_group_potential(hi2fa_test_frames):
+    _, sys_params, frames, boxes, ligand_idxs = hi2fa_test_frames
+    lamb = 0.0
+    num_atoms = frames.shape[1]
+    nonbonded_params = sys_params[3]
+    beta = 1 / (constants.BOLTZ * 300)
+    cutoff = 1.2
+
+    precisions = [np.float32, np.float64]
+
+    potential = NonbondedInteractionGroup(
+        ligand_idxs,
+        np.zeros(num_atoms, dtype=np.int32),
+        np.zeros(num_atoms, dtype=np.int32),
+        beta,
+        cutoff,
+    )
+    class_name = potential.__class__.__name__
+    for precision in precisions:
+        if precision == np.float32:
+            label = class_name + "_f32"
+        else:
+            label = class_name + "_f64"
```

Comment:
Could this logic be moved into the `benchmark_potential_over_*` functions to reduce duplication?

Reply:
Resolved in [3bf17ef](https://github.com/proteneer/timemachine/pull/741/commits/3bf17ef6f23380471ea241680180119ddbcc988e)